view/frontend/templates/js/replacejs.phtml

```diff
@@ -859,6 +862,9 @@ ob_start();
             checkout_buttons = true;
             processButtons();
         });
+        if (boltConfig.always_present_checkout) {
+            processButtons();
+        }
```

Comment:
when we have in cart both simple and configurable product and click “add to cart” (I do it for the same product that is already on cart)
When we check .block-minicart .subtitle it related to two different DOM nodes: subtitle of minicart and subtitle of configurable product
Function monitorDataChange thinks this was update and call createOrder() again and again.

If we delete lines 865-867 then after few attempts this reloading is finished.
I propose to remove them and completely fix the issue later when we refactor this JS file.

Reply:
Removed